sdk/rust/crates/dagger-sdk/src/core/mod.rs

```diff
@@ -1,4 +1,4 @@
-pub const DAGGER_ENGINE_VERSION: &'static str = "0.6.4";
+pub const DAGGER_ENGINE_VERSION: &'static str = "0.8.0";
```

Comment:
I don't think that this change is meant to be in this PR.

Reply:
I’m running the wrong bump. T_T Let me revert the change.

Reply:
Fix this issue in https://github.com/dagger/dagger/pull/5565/commits/7523310d8cb6a777aace0a283e1d1491a09f1942